pkg/scripts/bigchaindb-monit-config

```diff
@@ -129,21 +130,37 @@ exit 0
 EOF
 chmod +x ${monit_script_path}
 
+cat >${monit_script_path}_logrotate <<EOF
+#!/bin/bash
+case \$1 in
+
+  rotate_tendermint_logs)
+    /bin/mv \$2 \$2.\$(date +%y-%m-%d)
+    /bin/tar -cvf \$2.\$(date +%Y%m%d_%H%M%S).tar.gz \$2.\$(date +%y-%m-%d)
+    /bin/rm \$2.\$(date +%y-%m-%d)
+    /bin/kill -2 \$(cat \$3)
```

Comment:
I think, you can clear the file without removing it, then Tendermint does not have to be restarted. One way of clearing it is `cp /dev/null \$2`.